DirectX11/HackerContext.h

```diff
@@ -225,15 +211,6 @@ class HackerContext : public ID3D11DeviceContext1
 	virtual void FrameAnalysisDump(ID3D11Resource *resource, FrameAnalysisOptions options,
 		const wchar_t *target, DXGI_FORMAT format, UINT stride, UINT offset) {};
 
-	// These are the shaders the game has set, which may be different from
-	// the ones we have bound to the pipeline:
-	ID3D11VertexShader *mCurrentVertexShaderHandle;
-	ID3D11PixelShader *mCurrentPixelShaderHandle;
-	ID3D11ComputeShader *mCurrentComputeShaderHandle;
-	ID3D11GeometryShader *mCurrentGeometryShaderHandle;
-	ID3D11DomainShader *mCurrentDomainShaderHandle;
-	ID3D11HullShader *mCurrentHullShaderHandle;
-
```

Comment:
As stated above, shaders are assigned to per-context in DirectX11 (In DirectX10 they are per-device), so these absolutely should not be moved out of here - this will break games using multi-threaded rendering pipelines.